template/package.json

```diff
@@ -26,7 +26,7 @@
   "peerDependencies": {
```

Comment:
The commit name about removing focus visible doesn't seem to have anything to do with these changes.

Reply:
I am not following you? This comment is in the 'perf(review): add all suggested updates' commit? 

commit e717311

Reply:
I think I may have misclicked on a commit link and got confused.